System check identified some issues:

WARNINGS:
[33;1m?: (urls.W005) URL namespace 'anime' isn't unique. You may not be able to reverse all URLs in this namespace[0m
[{"model": "contenttypes.contenttype", "pk": 1, "fields": {"app_label": "admin", "model": "logentry"}}, {"model": "contenttypes.contenttype", "pk": 2, "fields": {"app_label": "auth", "model": "permission"}}, {"model": "contenttypes.contenttype", "pk": 3, "fields": {"app_label": "auth", "model": "group"}}, {"model": "contenttypes.contenttype", "pk": 4, "fields": {"app_label": "auth", "model": "user"}}, {"model": "contenttypes.contenttype", "pk": 5, "fields": {"app_label": "contenttypes", "model": "contenttype"}}, {"model": "contenttypes.contenttype", "pk": 6, "fields": {"app_label": "sessions", "model": "session"}}, {"model": "contenttypes.contenttype", "pk": 7, "fields": {"app_label": "hello", "model": "greeting"}}, {"model": "sessions.session", "pk": "w0ooi89c8ym6mn0m1fk8fmwttgctlra7", "fields": {"session_data": "MjMwYzRiNWRkZDM5MWJmMjk4YzYyZTUwMzMzNTE4M2M0OTliNzE2Yjp7Il9hdXRoX3VzZXJfaWQiOiIxIiwiX2F1dGhfdXNlcl9iYWNrZW5kIjoiZGphbmdvLmNvbnRyaWIuYXV0aC5iYWNrZW5kcy5Nb2RlbEJhY2tlbmQiLCJfYXV0aF91c2VyX2hhc2giOiI2NDNlOTkwMDBmZTU0MDIyNmEyN2YxYmIxY2U5YTJjM2Q3MjJlYjFkIn0=", "expire_date": "2019-12-01T00:22:45.058Z"}}[31;1mCommandError: Unable to serialize database: cursor "_django_curs_139677609103488_3" does not exist
[0m